github/github.go

```diff
@@ -154,6 +156,19 @@ type UploadOptions struct {
 	Name string `url:"name,omitempty"`
 }
 
+// RawType represents type of raw format of a request instead of JSON.
+type RawType int
+
+const (
+	Diff RawType = 1 + iota
+	Patch
+)
+
+// RawOptions specifies parameters when user want to get raw format of a response instead of JSON.
```

Comment:
nit: s/want/wants/
Also, if you browse around this repo, you will see that the comments attempt to stay within 80 columns although obviously this is not strictly necessary for Go programming. But just to keep things consistent, it might be nice to break this line.